lte/gateway/c/session_manager/SessionStateEnforcer.h

```diff
@@ -105,17 +137,27 @@ class SessionStateEnforcer {
 
   // Timer used to forcefully terminate session context on time out
   long session_force_termination_timeout_ms_;
-  bool static_rule_init();
+  bool default_and_static_rule_init();
+  // update the GNB endpoint details in a rule
+  bool insert_pdr_from_core(
+      const std::string& imsi, std::unique_ptr<SessionState>& session_state,
+      SessionStateUpdateCriteria& session_uc, SetGroupPDR& rule);

```

Comment:
Usually we tend to place any update criteria related fields at the very end.
```
bool insert_pdr_from_core(
      const std::string& imsi, std::unique_ptr<SessionState>& session_state,
      SetGroupPDR& rule, SessionStateUpdateCriteria& session_uc)
```

Reply:
Yes, modified it.